packages/manager/apps/telecom/src/app/telecom/pack/move/eligibility/address/move-eligibility-address.controller.js

```diff
@@ -137,6 +137,25 @@ export default class {
    * Copper eligibility by address
    */
   copperEligibilityByAddress() {
+    if (this.isReseller) {
+      // If current offer is a reseller offer,
+      // launch eligibility test address for partners (reseller)
+      return this.OvhApiConnectivityEligibility.v6()
+        .testAddressPartners(this.$scope, {
+          streetCode: this.address.street.streetCode,
+          streetNumber: this.address.streetNumber,
+        })
+        .then((res) => {
+          const elig = {
+            result: res.result,
+          };
+          return elig;
+        })
```

Comment:
Is this really needed ? As `res` seems to already have the result object you could return directly

Reply:
I just need result not all res